libraries/AP_Arming/AP_Arming.cpp

```diff
@@ -325,18 +325,36 @@ bool AP_Arming::ins_checks(bool report)
             check_failed(ARMING_CHECK_INS, report, "Gyros inconsistent");
             return false;
         }
+
+        // check AHRS attitudes are consistent
+        if (!AP::ahrs().attitudes_consistent()) {
+            check_failed(ARMING_CHECK_INS, report, "Attitudes inconsistent");
+            return false;
+        }
     }
 
     return true;
 }
 
 bool AP_Arming::compass_checks(bool report)
 {
+    Compass &_compass = AP::compass();
+
+    // check if compass is calibrating
+    if (_compass.is_calibrating()) {
+        check_failed(ARMING_CHECK_COMPASS, report, "Compass calibration running");
```

Comment:
Checks which can't be bypassed with a bit should say `ARMING_CHECK_NONE`, not specify a bit as is done here.

Reply:
txs, fixed in follow up commit